teams/teams-ps/teams/teams.md

```diff
@@ -56,6 +56,9 @@ The following cmdlet references are for Microsoft Teams.
 ### [Get-TeamsShiftsPolicy](Get-TeamsShiftsPolicy.md)
 {{Manually Enter Get-TeamsShiftsPolicy Description Here}}
 
+### [Get-TeamTargetingHierarchyStatus](Get-TeamTargetingHierarchyStatus.md)
+{{Manually Enter Get-TeamTargetingHierarchyStatus Description Here}}
```

Comment:
Since you're creating the cmdlets and providing descriptions through your other PRs, can you also add the short description here?